collect_app/src/main/java/org/odk/collect/android/activities/GoogleDriveActivity.java

```diff
@@ -789,6 +789,7 @@ void setTaskListener(TaskListener tl) {
                         .setFields("id")
                         .execute().getId();
             } catch (UserRecoverableAuthIOException e) {
+                createAlertDialog(getString(R.string.auth_failed_drive));
```

Comment:
I don't think that this is necessary. The next line pops up the auth request and I think it is clear enough what is required.